Automattic/newspack-plugin
# feat(ia): setup wizard

### All Submissions:


### Changes proposed in this Pull Request:

Add new IA wizards instantiations into `newspack_setup_complete` conditional. PR also includes minor fix for Newspack Dashboard slug.

### How to test the changes in this Pull Request:

1. Checkout this branch `git checkout origin/feat/ia-setup`
2. Delete `newspack_setup_complete` from db i.e. `wp option delete newspack_setup_complete`
3. Navigate to _/wp-admin/admin.php?page=newspack-setup-wizard_
![Screenshot 2024-12-12 at 09 43 15](https://github.com/user-attachments/assets/9eeea61c-6285-40d4-8a0d-c74d00d994be)
4. Confirm no other Newspack IA menus exist.
5. Complete Setup process. 
6. Once completed - refresh browser and observe new IA menus are present.

### Other information:
